highfidelity/hifi
# Adding haze component to the zone entity.

This PR adds a haze component to the Zone Entity.
The Haze component works similarly to the other visual effects components managed from  a zone (skybox, key light).

## TEST PLAN
- first install this PR interface  + sandbox
- run the PR Interface
- goto localhost PR sandbox
- pick an  environment with large enough scenery to see the haze effect ("desert oasis" for example)
- Edit the zone entity surrounding you (create, select the zone, properties)
- install a Haze component in the zone by picking the "on" choice from the 3 radio buttons (instead of inherit, the default)
- THe haze should show right away

A full test suite is available on TestRails, containing 7 cases.
https://highfidelity.testrail.net/index.php?/suites/view/126&group_by=cases:section_id&group_order=asc
## KNOWN ISSUES
- THere is a banding artifact in the color displayed in gradient variation of the color which is particularly visible with the Hze effect. This is also a different bug that we are trying to address.
